README.md

```diff
@@ -11,19 +11,12 @@ Welcome to the Sustainable Free and Open Source Community!
 This is the place to contribute content to the website, and collaborate
 on writing new social contracts, documenting business models, and more.
 
-The `master` branch is visible on https://sfosc.org - it's easier to read that way.
+**Find out how you can help us on the [how to help file](https://github.com/sfosc/sfosc/blob/master/HOW_TO_HELP.md).**
 
-To contribute, send a pull request. By doing so, you agree to have your content
-distributed and shared under the same terms as the LICENSE.md file dictates.
+The `master` branch is visible on https://sfosc.org - it's easier to read that way.
 
 You should read our [Social Contract](https://github.com/sfosc/sfosc/blob/master/SOCIAL_CONTRACT.md) - it contains everything you need to know about how the community works.
 
-If you want to join the project as a member (which pretty much just
-means you get to vote for the project leader), add yourself to the [membership file](https://github.com/sfosc/sfosc/blob/master/MEMBERSHIP.md).
-
-If you want to propose changes, or new directions for the project, start or join
-a thread in an issue.
-
 ## Sponsors
 
 Thanks to **[@adamhjk](https://github.com/adamhjk)** for sponsoring the `sfosc.org` domain.
```

Comment:
What about @Beanow for the Drone sponsorship?

Reply:
Good point, though separate issue. It's currently used only for https://github.com/sfosc/sourcecred, on sfosc/sfosc we use Drone's free cloud service.